src/containers/WelcomePage/WelcomePage.tsx

```diff
@@ -6,16 +6,22 @@
  *
  */
 
-import React, { Fragment } from 'react';
+import React, { FC, Fragment, useEffect, useState } from 'react';
+import BEMHelper from 'react-bem-helper';
+//@ts-ignore
```

Comment:
Er ikke så fan av den haha, er det noe måte å få det fikset eller var det det at da måtte man omskrive alle filene om til .tsx?

Reply:
Mener du med ts-ignore? 
Det var løsningen Jonas kom med, og som vi fant ulike steder i koden. Problemet var at filen ikke fant OneColumn fordi den ikke var TypeScript eller noe av den typen. Forstod det slik at man ikke kan bruke JavaScript komponent i komponent som bruker TypeScript. Så for å unngå å skrive om kode jeg ikke kjenner til og som kan være avhengig av mye kode så var dette raskeste løsningen vi fant. Vet ikke om det finnes andre løsninger, kan google litt og se.

Reply:
Ja jeg mente ts-ignore, men hvis du må endre alle filer om til .tsx så er det litt mye jobb. La det stå.